### DISCUSSION TITLE
treat ^h (ctrl+h) as backspace

### DISCUSSION THREAD

**Discussion by p00ya** [monkeytypegame/monkeytype#1758] [Ideas] [answered]
macOS, bash and UNIX users will be familiar with the ubiquitous ^h code for deleting backwards a single character (it works just the same as the backspace key, because historically the backspace key actually sent this keycode).  Some references:

- https://support.apple.com/en-us/HT201236 (under "Document shortcuts")
- https://en.wikipedia.org/wiki/Backspace#^H

Any native text field (or indeed Chrome/Safari text field unless they're doing special input processing) will treat ^h as backspace on macOS.  Monkeytype ignores ^h.

It would be good for monkeytype to treat ^h as backspace either by default or via opt-in.  It's typically much more ergonomic (e.g. H is on the home row right index finger on qwerty/dvorak/colemak) and it's quite frustrating to have the muscle-memory not work on monkeytype.

I can send a PR if this seems acceptable.  Please let me know if it should be configurable/opt-in.

**Comment by Miodec** [comment, marked as answer]
#1325 should fix this completely
